src/sentry/incidents/endpoints/serializers.py

```diff
@@ -59,6 +59,41 @@
 CRITICAL_TRIGGER_LABEL = "critical"
 WARNING_TRIGGER_LABEL = "warning"
 
+# TODO: This is temporarily needed.
+# Use a function from discover to break the aggregate down into parts, and then compare the "field"
+# to a list of accepted fields, or a list of fields we need to translate.
+# This can be dropped once snuba can handle this aliasing.
+SUPPORTED_COLUMNS = [None, "tags[sentry:user]", "transaction.duration"]
+TRANSLATABLE_COLUMNS = {
+    "user": "tags[sentry:user]",
+    "dist": "tags[sentry:dist]",
+    "release": "tags[sentry:release]",
+}
+
+
+def forward_translate_snuba_field(aggregate):
+    field = resolve_field(aggregate)
+    column = field[1][0][1]
+    if column in SUPPORTED_COLUMNS:
+        return aggregate
+    elif column in TRANSLATABLE_COLUMNS.keys():
+        return aggregate.replace(column, TRANSLATABLE_COLUMNS[column])
```

Comment:
Doing a string replace here makes me a little nervous, but I assume we probably won't have function names that are the same as cols. I'd rather reassemble the function from components and just replace the aggregate, but not sure if you end up getting all that info here.